src/Symfony/Component/Messenger/Tests/Stamp/ErrorDetailsStampTest.php

```diff
@@ -38,11 +39,24 @@ public function testUnwrappingHandlerFailedException(): void
         $exception = new HandlerFailedException($envelope, [$wrappedException]);
         $flattenException = FlattenException::createFromThrowable($wrappedException);
 
-        $stamp = new ErrorDetailsStamp($exception);
+        $stamp = ErrorDetailsStamp::create($exception);
 
         $this->assertSame(\Exception::class, $stamp->getExceptionClass());
         $this->assertSame('I am inside', $stamp->getExceptionMessage());
         $this->assertSame(123, $stamp->getExceptionCode());
         $this->assertEquals($flattenException, $stamp->getFlattenException());
     }
+
+    public function testDeserialization(): void
+    {
+        $exception = new \Exception('exception message');
+        $stamp = ErrorDetailsStamp::create($exception);
+        $serializer = new Serializer();
```

Comment:
The Messenger component registers a [normalizer](https://github.com/symfony/symfony/blob/5.x/src/Symfony/Component/Messenger/Transport/Serialization/Normalizer/FlattenExceptionNormalizer.php) to do it right.

Test ref: https://github.com/symfony/symfony/blob/5.x/src/Symfony/Component/Messenger/Tests/Transport/Serialization/Normalizer/FlattenExceptionNormalizerTest.php

Reply:
So I should change the test adding the normalizer? Or should I just ignore those differences?

Reply:
Yes, you should change the test accordingly to use the normalizer and the context needed.

Reply:
I've added the normalizers, but now the `statusText` property is still missing from the normalization.. what am I missing now?

Reply:
It's missing the context:
https://github.com/symfony/symfony/blob/77c82cfa18fc70bf3d57f244d8338be04fea3774/src/Symfony/Component/Messenger/Transport/Serialization/Normalizer/FlattenExceptionNormalizer.php#L98

Reply:
@yceruto that's already the default: https://github.com/symfony/symfony/blob/78fbd0ac77a571afc332938a5ec836e986de5214/src/Symfony/Component/Messenger/Transport/Serialization/Serializer.php#L44

Reply:
Ok, that means the normalizer needs to be updated accordingly. Could you have a look?

Reply:
Fixed!

Reply:
Fixed!